src/test/java/net/datafaker/annotations/FakeAnnotationTest.java

```diff
@@ -0,0 +1,56 @@
+package net.datafaker.annotations;
+
+import java.util.Locale;
+import java.util.Random;
+
+import net.datafaker.Faker;
+import net.datafaker.annotations.dto.Person;
+import net.datafaker.service.RandomService;
+import net.datafaker.transformations.Schema;
+import org.junit.jupiter.api.Test;
+
+import static net.datafaker.transformations.Field.field;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FakeAnnotationTest {
+
+    @Test
+    void shouldGenerateEntityWithDefaultSchema() {
+        var person = Faker.populate(Person.class);
+
+        assertThat(person).isNotNull();
+        assertThat(person.getFullName()).isEqualTo("Dr Alexis Noël");
+    }
+
+    @Test
+    void shouldGenerateEntityWithCustomSchema() {
+        var person = Faker.populate(Person.class, customSchema());
+
+        assertThat(person).isNotNull();
+        assertThat(person.getFullName()).isEqualTo("Dr Alexis Noël");
+    }
+
+    @Test
+    void shouldGenerateEntityWithDefaultSchemaAndInDefaultSchemaInCurrentClass() {
+        var person = Faker.populate(DefaultPerson.class);
+
+        assertThat(person).isNotNull();
+        assertThat(person.fullName).isNotNull();
+    }
+
+    public static Schema<Object, ?> defaultSchema() {
+        var faker = new Faker(Locale.forLanguageTag("fr-en"), new RandomService(new Random(1)));
+        return Schema.of(field("fullName", () -> faker.name().fullName()));
+    }
+
+    public static Schema<Object, ?> customSchema() {
+        var faker = new Faker(Locale.forLanguageTag("fr-en"), new RandomService(new Random(1)));
+        return Schema.of(field("fullName", () -> faker.name().name()));
+    }
```

Comment:
It would be better to have custom and default schemas different since if you look at implementation of `Name#name`
```java
    public String fullName() {
        return name();
    }
```
they are the same =)